src/components/structures/IndicatorScrollbar.js

```diff
@@ -127,20 +120,19 @@ export default class IndicatorScrollbar extends React.Component {
 
     onMouseWheel = (e) => {
         if (this.props.verticalScrollsHorizontally && this._scrollElement) {
-            const xyThreshold = this.props.scrollTolerances
-                ? this.props.scrollTolerances.xyThreshold
-                : Number.MAX_SAFE_INTEGER;
-
-            const yReduction = this.props.scrollTolerances
-                ? this.props.scrollTolerances.yReduction
-                : 0.8;
-
-            // Don't apply vertical motion to horizontal scrolls. This is meant to eliminate
-            // trackpads causing excessive scroll motion.
-            if (e.deltaX >= xyThreshold) return;
-
-            // noinspection JSSuspiciousNameCombination
-            this._scrollElement.scrollLeft += e.deltaY * yReduction;
+            // xyThreshold is the amount of horizontal motion required for the component to
+            // ignore the vertical delta in a scroll. Used to stop trackpads from acting in
+            // strange ways. Should be positive.
+            const xyThreshold = 0;
```

Comment:
Interesting choice to keep the variables around when they're both effectively causing no change... I guess it's in case we want to adjust down the road? Would it be simpler to just remove for now?

Reply:
It's more to keep the documentation for the behaviour, and to make it slightly easier to tweak down the road if needed.